documentation/3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/3_4_InformationLayer.md

```diff
@@ -0,0 +1,224 @@
+that can be traded and exchanged between remote participants using the
+IDS infrastructure. Following the web resource paradigm[^11], the
+abstract content of a Resource is provided in a variety of
+representations. Examples of Resources are documents, time series of
+sensor values, messages, image file archives, or media streams.
+Resources are subject to forwarding, processing, and/or consumption,
+with a particular demand for modeling related, complementary aspects
+(i.e., content, provenance, provisioning etc.). These are analyzed and
+specified here by applying the separation of concerns (SoC)
+paradigm[^12].
+
+#### Separation of Concerns (SoC)
+
+Following the separation of concerns design principle, only one
+dimension of a subject matter is considered at a time, for the sake of
+clarity and consistency. Similar to the principle a microscope works,
+each concern follows a particular, analytical point of view, while other
+concerns can temporarily be disregarded. This principle can be applied
+to information modeling, aiming at a thorough understanding of the
+domain and fostering modularity and re-usability of the resulting (sub-)
+models. Accordingly designed, these models may evolve independently of
+each other and can be updated by different agents at different times. As
+any modification of a single element of the overall model does not
+require a change in other, logically unrelated parts, the development
+and maintenance of models can be substantially simplified.
+
+::: Center
+![Outline of the Concern-Basic concern
+hexagon](./media/image32.png){#fig:Outline_of_the_ConcernBasic_concern_hexagon
+width="3.64in" height="2.67in"}
+:::
+
+#### Concern Hexagon
+
+To illustrate the main modeling [c]{.underline}oncerns of Digital
+Resources in a way easy to memorize, the mnemonic hexagonal arrangement
+of [c]{.underline}arbon atoms can be used (C-Hexagon), as shown in
+Figure
+[2](#fig:Outline_of_the_ConcernBasic_concern_hexagon){reference-type="ref"
+reference="fig:Outline_of_the_ConcernBasic_concern_hexagon"}. As a
+Resource's content is its most essential aspect, Content is located at
+the top of the hexagon. This content is interpretable by references to a
+shared, formally defined Concept, whereas links to a particular
+*C*ontext (in terms of time, place, or real-world entities) make the
+content potentially relevant for certain Data Consumers. So the upper
+part of the C-Hexagon deals with the what aspects, independently of
+Data Exchange, Data Sharing or Data Utilization. The lower part relates
+to the how aspects; i.e. how the content is exchanged
+(*C*ommunication) and under which conditions (*C*ommodity). The
+*C*ommunity of Trust concern refers to the distinctive feature of the
+International Data Spaces being an ecosystem of certified participants
+and components that exchange and share Digital Resources in accordance
+with usage policies ensuring data sovereignty.
```

Comment:
```suggestion
operating certified components, such as Connectors. Using such components, Participants exchange and share Digital Resources in a secure and trusted way in accordance
with contracts composed of usage policies, thus ensuring data sovereignty.
```
Now spending one paragraph on each concern, reintroducing revised versions of the summaries we had in the RAM 3.0.